### PR TITLE
Migrate Gradle to Version Catalog, bump Gradle to 8.0

### PR DESCRIPTION
**Pull request by chiragthummar**
I have just migrated gradle to Version Catalog with Kotlin DSL

### CODE REVIEW

**Comment by Bnyro** [comment]
Looks like a great way of moving the app forward, thanks for your work!

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (Bnyro) [inline gradle/libs.versions.toml:69]: I guess we don't need all those blank lines?

Comment (chiragthummar) [reply gradle/libs.versions.toml:69]: Yeah, You are right

**Comment by Bnyro** [comment]
Thank you!

**Pull request merged**